src/test/java/hu/akarnokd/rxjava/interop/RxJavaInteropV2SchedulerToV1SchedulerTest.java

```diff
@@ -0,0 +1,173 @@
+package hu.akarnokd.rxjava.interop;
+
+import io.reactivex.Scheduler;
+import io.reactivex.schedulers.TestScheduler;
+import org.junit.Test;
+import rx.functions.Action0;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.*;
+
+public class RxJavaInteropV2SchedulerToV1SchedulerTest {
+
+    @Test
+    public void now() {
+        Scheduler v2Scheduler = mock(Scheduler.class);
```

Comment:
Can't we use the normal `TestScheduler` here for testing? All of this white box mockito testing can be really brittle

Reply:
A comment to remove Mockito usage? Instant Jake-thumbs-up to that: 👍

Reply:
Also not a fan of Mockito and mocking, however in scenarios like proxying which is what I do here it's saves us from boilerplate and keeps code readable

Especially in few tests below when I need to verify invocations